### PR TITLE
Update 1.0.0

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

File: recipe/meta.yaml
Comment (jezdez) [inline recipe/meta.yaml:46]: Should this be pinned to a specific version?

Comment (katietz) [reply recipe/meta.yaml:46]: yes, as we want to define here a minimal version.  a >= would pick always a max version, which is not necessarily what you wanted here

File: recipe/meta.yaml
Comment (jezdez) [inline recipe/meta.yaml]: Should we reset that to 0?

File: recipe/meta.yaml
Comment (jezdez) [inline recipe/meta.yaml]: I'm wondering if this was legitimately fixed? I think this was added by Anaconda staff in the past.

File: recipe/meta.yaml
Comment (jezdez) [inline recipe/meta.yaml:95]: I don't understand the difference, I don't see a file size difference

File: recipe/meta.yaml
Comment (jezdez) [inline recipe/meta.yaml]: Do we need those for building the Python package?

**Review by jezdez — COMMENTED** [review]
Just a few comments @katietz, thank you for your work!

**Comment by jezdez** [comment]
@jaimergp Would you mind taking a look at this?

File: recipe/meta.yaml
Comment (jaimergp) [inline recipe/meta.yaml]: Shouldn't this be >=0.7.22 (as in host)?

```suggestion
        - libsolv >=0.7.22
```

**Comment by jezdez** [comment]
Should we rename the repo to mamba-feedstock to denote change in strategy?

**Comment by jezdez** [comment]
@katietz Is there anything left on your side for this?

**Comment by katietz** [comment]
no, fixes have a PR on CF too, otherwise I just wait for your approval on it

**Review by jezdez — APPROVED** [review]

**Comment by katietz** [comment]
Merge it with just one approver, but actual there was a second-one involved, so it is fine

**Pull request merged**
